counsel.el

```diff
@@ -2699,6 +2719,34 @@ Note: don't use single quotes for the regex."
 (ivy-set-occur 'counsel-rg 'counsel-ag-occur)
 (ivy-set-display-transformer 'counsel-rg 'counsel-git-grep-transformer)
 
+(defcustom counsel-rg-pcre-flag 'auto
+  "PCRE flag used by Ripgrep.
+
+If `auto', flag will be determined automatically.
+
+If `nil', PCRE support is disabled unless \"--pcre2\" or \"-P\"
```

Comment:
No need to quote `nil` and `t` in docstrings; see [`(elisp) Documentation Tips`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html).